<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrollGrid</class>
 <widget class="QDialog" name="ScrollGrid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    border-image:url(:/files/theme16Ñ…9.jpg);
    width: 100px;
    border: none;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background: transparent; border: none;</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>990</width>
        <height>637</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>0</height>
      </size>
     </property>
     <property name="pageStep">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
